src/mscorlib/src/System/Globalization/TextInfo.Windows.cs

```diff
@@ -49,44 +41,44 @@ private unsafe string ChangeCase(string s, bool toUpper)
             {
                 return s;
             }
-            else
+
+            int ret;
+
+            // Check for Invariant to avoid A/V in LCMapStringEx
+            uint linguisticCasing = IsInvariantLocale(_textInfoName) ? 0 : LCMAP_LINGUISTIC_CASING;
+
+            //
+            //  Create the result string.
+            //
+            string result = string.FastAllocateString(nLengthInput);
+
+            fixed (char* pSource = s)
+            fixed (char* pResult = result)
+            {
+                ret = Interop.Kernel32.LCMapStringEx(_sortHandle != IntPtr.Zero ? null : _textInfoName,
+                                                    toUpper ? LCMAP_UPPERCASE | linguisticCasing : LCMAP_LOWERCASE | linguisticCasing,
+                                                    pSource,
+                                                    nLengthInput,
+                                                    pResult,
+                                                    nLengthInput,
+                                                    null,
+                                                    null,
+                                                    _sortHandle);
+            }
+
+            if (0 == ret)
```

Comment:
As you know in native code people often write `constant == variable` (aka Yoda conditionals) out of habit so that if they mistype `constant = variable` it will cause a compile error. In C# though there is no implicit conversion in most cases to boolean so eg for integers there is no danger of successfully compiling `constant = variable`. So in code bases I"ve worked on, typically it's written `variable == constant` for readability.
Just a nit and I don't see it in the style guidelines although it does seem largely followed here.